### DISCUSSION TITLE
Why graphin doesn't comply with g6 API args?

### DISCUSSION THREAD

**Discussion by shivamka1** [antvis/Graphin#494] [Q&A]
I was going through this custom highlight feature and thought of trying it using graphin library.
It turns out `graph.setItemState(edge.getTarget(), 'highlight', true);` doesn't take any effect!? I had to try other string arguments such as `'active'`, `'disabled'` to get this working as mentioned [here](https://github.com/antvis/Graphin/blob/725cfabee632b6bdefd6f60b65aee5710299bff4/packages/graphin/docs/render/status/index.md?plain=1#L21).

My question is why graphin doesn't uses the same arguments as g6, if that is indeed the case, unless I have misunderstood it. 

---

我正在经历这个自定义突出显示功能，并考虑使用 graphin 库来尝试它。
事实证明 `graph.setItemState(edge.getTarget(), 'highlight', true);` 没有任何效果！？我必须尝试其他字符串参数，例如 `“active”`、`“disabled”` 才能使其按照此处提到的方式工作，或者我相信是这样（它是中文的）。

我的问题是为什么 graphin 不使用与 g6 相同的参数，如果确实如此，除非我误解了它

**Comment by pomelo-nwu** [comment]
Because Graphin has expanded the status based on the custom mechanism of G6：https://github.com/antvis/Graphin/blob/master/packages/graphin/src/theme/node-style.ts#L112
